packages/block-library/src/navigation-submenu/edit.js

```diff
@@ -284,6 +284,7 @@ export default function NavigationSubmenuEdit( {
 	const { label, type, opensInNewTab, url, description, rel, title, kind } =
 		attributes;
 	const link = {
+		title: label,
```

Comment:
I think this will need to be escaped before it's provided. We have `escapeHTML` from `@wordpresss/escape-html` for this.

In the longer run we can normalize all this code with the `LinkUI` component from `core/navigation-link`.

Reply:
```suggestion
		title: label && stripHTML( label ),
```

Sorry I meant `stripHTML`.